apps/docs/content/docs/components/table.mdx

```diff
@@ -149,8 +149,11 @@ provide a default set of selected rows.
 
 <CodeDemo title="Single Row Selection" files={tableContent.singleSelection} />
 
-> **Note**: The value of the selected keys must match the key prop of the row.
+> **Note**:
+> - The value of the selected keys must match the `key` prop of the row.
+> - React keys are automatically converted to strings. When using the `Table` component, ensure that all keys (e.g., row IDs) are treated as strings to avoid type mismatch issues. For more information, refer to the [React documentation on keys](https://softwareengineering.stackexchange.com/questions/381268/why-are-react-keys-limited-to-strings)
```

Comment:
1. `refer to the [React documentation on keys]`: That's not official React documentation.
2. `all keys (e.g., row IDs)`: change to `all row keys` instead.
3. Change the link `https://softwareengineering.stackexchange.com/questions/381268/why-are-react-keys-limited-to-strings/432948#432948` instead.

Reply:
@abhinandan-verma You haven't addressed 1 - 3 at all.